packages/widget-v2/src/components/Button.tsx

```diff
@@ -28,6 +28,37 @@ export const GhostButton = styled(SmallText).attrs({
   ${flexProps};
 `;
 
+export const TransparentButton = styled(SmallText).attrs({
+  as: "button",
+}) <GhostButtonProps>`
+  ${removeButtonStyles};
+  line-height: 13px;
+  ${({ theme, onClick, secondary, disabled }) =>
+    onClick &&
+    !disabled &&
+    css`
+      background-color: ${secondary
+        ? theme.secondary.background.normal
+        : theme.primary.ghostButtonHover};
+      cursor: pointer;
+    `};
+  &:hover {
+    ${({ theme, onClick, secondary, disabled }) =>
+    onClick &&
+    !disabled &&
+    css`
+        background-color: ${secondary
+        ? theme.secondary.background.normal
+        : theme.primary.ghostButtonHover};
+        color: ${theme.primary.text.normal};
+        cursor: pointer;
+      `};
+  }
+  padding: 10px 13px;
+  border-radius: 90px;
+  ${flexProps};
+`;
+
```

Comment:
I would suggest:

1. Move this to SwapPage.tsx since this isn't a generic button
2. TransparentButton should be based on GhostButton like this and it will inherit all of GhostButton styles so you don't need to duplicate them

```ts
const TransparentButton = styled(GhostButton)``
```